tests/ansible/regression/all.yml

```diff
@@ -1,18 +1,5 @@
-- import_playbook: issue_109__target_has_old_ansible_installed.yml
-- import_playbook: issue_113__duplicate_module_imports.yml
-- import_playbook: issue_118__script_not_marked_exec.yml
-- import_playbook: issue_122__environment_difference.yml
-- import_playbook: issue_140__thread_pileup.yml
-- import_playbook: issue_152__local_action_wrong_interpreter.yml
-- import_playbook: issue_152__virtualenv_python_fails.yml
-- import_playbook: issue_154__module_state_leaks.yml
-- import_playbook: issue_177__copy_module_failing.yml
-- import_playbook: issue_332_ansiblemoduleerror_first_occurrence.yml
-- import_playbook: issue_558_unarchive_failed.yml
-- import_playbook: issue_590__sys_modules_crap.yml
-- import_playbook: issue_591__setuptools_cwd_crash.yml
-- import_playbook: issue_615__streaming_transfer.yml
```

Comment:
🤦 I left some WIP deletions in.

Reply:
Corrected in 1d319f3d, which I've pushed to #1220. Sorry for the noise.

Reply:
Sorted, no worries :)